packages/playground-preview-worker/wrangler.json

```diff
@@ -17,7 +17,7 @@
 
 	"define": {
 		"ROOT": "'playground/devprod/cloudflare/dev'",
-		"PREVIEW": "':any/playground/devprod/cloudflare/dev'"
+		"PREVIEW": "':any/cloudflarepreviews/com'"
```

Comment:
out of curiosity, why are we using `define` macros here instead of env vars?

Reply:
Because the values are used for setting up Hono's router, which happens outside of the request context